compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -2573,11 +2582,11 @@ impl<'a> Parser<'a> {
     }
 
     pub(super) fn parse_arm(&mut self) -> PResult<'a, Arm> {
-        fn check_let_expr(expr: &Expr) -> (bool, bool) {
+        fn check_let(expr: &Expr) -> (bool, bool) {
```

Comment:
nit: why rename this?

Reply:
I created this function but `let` is not an expression, thus the renaming.

It is better to revert because the change is not related to the PR.